keps/sig-node/0009-node-heartbeat.md

```diff
@@ -228,6 +224,20 @@ relying on frequent Node object updates. However, in non-managed solutions, cust
 will still be able to restore previous behavior by setting appropriate flag values.
 Thus, changing defaults to what we recommend is the path to go with.
 
+To evaluate the risk of further reduction of NodeStatus updates, I went through
+couple thousands of files mentioning `LastHeartbeatTime` and `k8s` on github to
+understand the potential of breaking some system. There are couple places that
+are incompatible with node leases, but they would already be broken by current
+1m frequency, e.g.:
```

Comment:
The Mesos Framework is deprecated, but would be nice to file issues on the other repos, if possible, to let them know about this KEP.

Reply:
Filed (couldn't file the one for mesos, as the repo is already in read-only mode).